Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```

Comment:
Wouldn't it be better to lower the required Python version to something like 3.7? Since we don't use 3.8 and 3.9 features.

```suggestion
python_version = "3.7"
```